seqr/utils/file_utils.py

```diff
@@ -64,3 +64,25 @@ def _google_bucket_file_iter(gs_path, byte_range=None, raw_content=False, user=N
             line = line.decode('utf-8')
         yield line
 
+
+def get_vcf_filename(vcf_path):
+    if vcf_path.endswith('.vcf') or vcf_path.endswith('.vcf.gz') or vcf_path.endswith('.vcf.bgz'):
+        return vcf_path
+    process = subprocess.Popen('gsutil ls ' + vcf_path, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=True)
+    if process.wait() != 0:
+        return None
+    for line in process.stdout:
+        file = line.decode('utf-8').strip()
+        if file.endswith('.vcf') or file.endswith('.vcf.gz') or file.endswith('.vcf.bgz'):
```

Comment:
if we are want to validate that a file has a specific file extension, we should be validating that well upstream of this check, we should be validating that before we even check if the file exists and should throw an explicit error stating that the file extension is not supported